docs/devsetup-netex.md

```diff
@@ -0,0 +1,11 @@
+### How to setup netex conversion in development environment
```

Comment:
Ok, but if you like we could mention the netex unit test as well in section 3

Reply:
👍